app/src/main/java/com/account/persistence/entity/Account.java

```diff
@@ -0,0 +1,91 @@
+package com.account.persistence.entity;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import com.account.model.AccountDTO;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name= "account")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Account {
+    
+    @Id
+    @GeneratedValue
+    private UUID id;
+    
+    private String email;
+
+    private String username;
+
+    private String password;
+
+    private String status;
+
+    private LocalDateTime create_at;
+
+
```

Comment:
Missed updated_at
`updated_at`, timestamp(no time zone), configure entity to update on every write operation